app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java

```diff
@@ -719,6 +721,14 @@ private void onSelectEntriesResult(int resultCode, Intent data) {
             return;
         }
 
+        boolean wipeEntries = data.getBooleanExtra("wipeEntries", false);
+        if (wipeEntries) {
+            List<VaultEntry> entriesToBeRemoved = new ArrayList<>(_vault.getEntries());
```

Comment:
Perhaps it would make sense to add a ``clear()`` method to the ``UUIDMap`` class and call through to that in ``VaultManager``, so that we don't have to create a copy of the entry list and remove them one by one here.